docs/sources/dashboards/dashboard-public.md

```diff
@@ -0,0 +1,48 @@
+---
+aliases:
+  - /docs/grafana/latest/dashboards/dashboard-manage/
+  - /docs/grafana/latest/features/dashboard/dashboards/
+title: Public dashboards
+weight: 8
+---
+
+## Public dashboards
+
+> **Note:** This is an opt-in alpha feature.
+
+> **Caution:** Making your dashboard public could result in a large number of queries to the datasources used by your dashboard.
+> This can be mitigated by utilizing the enterprise [caching](https://grafana.com/docs/grafana/latest/enterprise/query-caching/) and/or rate limiting features.
+
+Public dashboards allow you to share your Grafana dashboard with anyone. This is useful when you want to expose your
+dashboard to the world.
+
+#### Enable the feature
+
+Add the `publicDashboards` feature toggle to your `custom.ini` file.
+
+> **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
+
+#### Make a dashboard public
+
+- Click on the sharing icon to the right of the dashboard title.
+- Click on the Public Dashboard tab.
+- Acknowledge the implications of making the dashboard public by checking all the checkboxes.
+- Turn on the Enabled toggle.
+- Click `Save Sharing Configuration` to make the dashboard public and make your link live.
+- Copy the public dashboard link if you'd like to share it. You can always come back later for it.
+
+#### Revoke access
+
+- Click on the sharing icon to the right of the dashboard title.
+- Click on the Public Dashboard tab.
+- Turn off the Enabled toggle.
+- Click `Save Sharing Configuration` to save your changes.
+- Anyone with the link will not be able to access the dashboard publicly anymore.
+
+#### Limitations
```

Comment:
Should we add a note about the security model (similar to the one we have for [anonymous dashboard access](https://grafana.com/docs/grafana/next/setup-grafana/configure-security/#implications-of-enabling-anonymous-access-to-dashboards))? It's hard to figure out the risks of enabling the feature based on the content of this doc.

Reply:
Yeah, if the risks aren't immediately apparent, then 100%. I'll add the section at the top after the intro

Reply:
looks great!:)